### PR TITLE
Made mock UI tests credentials shorter.

### PR DESCRIPTION
**Pull request by pachlava**
### Description
Entering text is one of the slowest parts of the UI tests workflow currently, especially in the latest Xcode 13.1 / iPhone 13 setup.

User email and password are two strings that are entered before every UI test, and the current values are not that short. These credentials are accepted by WireMock server, which does not care about their content (unless we make it do so), so we can make them shorter.

This change makes the login process 8 seconds faster.

P.S. There's no real user with such credentials, so we don't lose anything by changing them and "forgetting" about `e2eflowtestingmobile@example.com`. If it existed, it would make sense to keep them, because it simplifies mocks maintenance a lot.

### Test
- CI is green

### CODE REVIEW

**Review by jostnes — COMMENTED** [review]
I noticed this too, entering text is quite slow on the simulator. I was wondering if the speed is configurable but looks like it's not. Thanks for taking the initiative to update the length! 

One thing that I usually do for text fields is to include one alphabet, one number and one special character on so that we know the field can accept these types. But if Wire Mock doesn't care about the input value then it's not necessary. Seems like it also doesn't care if the password is 2 characters long (I checked on WP from a browser, when updating password, the minimum character is 6). 

To be honest, I'm not sure if it's a bug or is it functioning as designed when using Wire Mock (values defined in this file `WPUITestCredentials` will be assumed to be the correct value even if not meeting the password requirement)?

**Review by mokagio — APPROVED** [review]
@jostnes makes some good points about coverage in [her comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/17700#pullrequestreview-840170264).

I think we could write dedicated UI tests for details such has the kind of characters the email text input accepts and the minimum password length.

Given that these are the default values that we put in on the setup phase up of most flows, I like the implementation here that prioritizes speed. It's a 👍 for me.

**Comment by mokagio** [comment]
@pachlava I'm going to bump this to the next release because we'll be code freezing 19.0 today. Unlike #17701, which I just merged, this PR has one comment and one approval review, and I'd like @jostnes to give an explicit approval before merging this. 🙂

**Comment by pachlava** [comment]
> To be honest, I'm not sure if it's a bug or is it functioning as designed when using Wire Mock (values defined in this file WPUITestCredentials will be assumed to be the correct value even if not meeting the password requirement)?

This is a good point, and it opens quite a broad topic. I think there can be several possibilities (keeping in mind that we talk here about "existing" credentials, not the signup process):
1. If the only task of the app is to pass the existing password to API for validity check, it automatically becomes out of our scope (which is UI tests automation). Under WireMock, WM starts acting as an API, and so far it allows any password (i.e. we defined the mock file this way).
2. It's even possible that the actual check for existing password validity might not happen anywhere in real API, because it expects that the password meets the length/content criteria by default, because it was ensured during signup.
3. The app itself should check for password length/content before sending it to API. This is not happening now, otherwise this PR would not be functional. We can open a discussion in the dev channel if it makes sense to implement this check inside the app.

**Review by jostnes — APPROVED** [review]
Ah sorry just saw this. Approving this to get the speed improvement benefits from the change. The password requirement/validation part can be discussed separately.

**Pull request merged**
